packages/f-header/src/components/Navigation.vue

```diff
@@ -144,6 +145,44 @@
                         {{ help.text }}
                     </a>
                 </li>
+
+                <template v-if="isBelowMid">
```

Comment:
```suggestion
                <template v-else>
```
`v-if` and `v-else` makes easier to read

Reply:
If you're going to use if/else could you move this block to the top so the if check is against `isBelowMid` and not `!isBelowMid`, please.

Reply:
✅